cime_config/config_component.xml

```diff
@@ -210,6 +210,7 @@
       <value compset="^SSP370_"	                    >1850-2100_SSP3-7.0_transient</value>
       <value compset="^SSP534_"	                    >1850-2100_SSP5-3.4_transient</value>
       <value compset="^SSP245_"	                    >1850-2100_SSP2-4.5_transient</value>
+      <value compset="^HIST_ne0np4"                 >1850-2100_SSP2-4.5_transient</value>
```

Comment:
@ekluzek I can update this if you tell me that:
1) I put this in the wrong place
2) You think we need one line per VR grid rather than referring to all three as ne0np4

Reply:
This is the correct place, but does need to be three lines and slightly changed...

Reply:
``` xml
      <value compset="^HIST" grid="ne0np4.ARCTICGRIS.ne30x8" >1850-2100_SSP2-4.5_transient</value>
      <value compset="^HIST" grid="ne0np4.ARCTIC.ne30x8" >1850-2100_SSP2-4.5_transient</value>
```

And same for conus. Try that and make sure it works...

Reply:
From meeting with @ekluzek 
...or instead of 3 lines, set grid="ne0np4" in a single line for all three.

Reply:
The three tests that I submitted on izumi built. They are in the overnight queue now and should run after 6 pm MT:
```
./create_test SMS_Ln9.ne0ARCTICne30x4_ne0ARCTICne30x4_mt12.IHistClm60Sp.izumi_intel.clm-clm60cam6LndTuningMode_1979Start
./create_test SMS_Ln9.ne0ARCTICGRISne30x8_ne0ARCTICGRISne30x8_mt12.IHistClm60Sp.izumi_intel.clm-clm60cam6LndTuningMode_1979Start
./create_test SMS_Ln9.ne0CONUSne30x8_ne0CONUSne30x8_mt12.IHistClm60Sp.izumi_intel.clm-clm60cam6LndTuningMode_2013Start
```

Reply:
The tests fail because I have not **./rimport**ed the new files to the repository, yet. 
I will run ctsm_sci on derecho when the machine becomes available.

Reply:
ctsm-sci test-suite IN PROGRESS

Reply:
ctsm-sci OK I think, but confirming with @samsrabin if you're the right person to ask:
The only "unexpected" result is
`FAIL RXCROPMATURITY_Lm61.f09_g17.IHistClm50BgcCrop.derecho_intel.clm-cropMonthOutput RUN time=4 (UNEXPECTED: expected PEND)`
The error says
```
subprocess.CalledProcessError: Command '. /glade/derecho/scratch/slevis/tests_0507-183812de/RXCROPMATURITY_Lm61.f09_g17.IHistClm50BgcCrop.derecho_intel.clm-cropMonthOutput.C.0507-183812de_int.gddgen/.env_mach_specific.sh; CONDA_PREFIX=;  conda run -n npl python3 /glade/work/slevis/git/mksurfdata_toolchain/tools/modify_input_files/fsurdat_modifier /glade/derecho/scratch/slevis/tests_0507-183812de/RXCROPMATURITY_Lm61.f09_g17.IHistClm50BgcCrop.derecho_intel.clm-cropMonthOutput.C.0507-183812de_int.gddgen/modify_fsurdat_allcropseverywhere.cfg ' returned non-zero exit status 1.
```
My PR touched namelist_defaults, cime_config code, and mksurfdata_esmf code, so I would not expect it to affect or cause this error. Let me know what you think.

Reply:
Actually, would you mind adding that test to the expected fail list?

Reply:
@slevis-lmwg that test should be in the expected fail list. So add it if not. I thought I added it though...

Reply:
Ok, thanks. I will change the expected failure "status" from PEND to FAIL.